### PR TITLE
nit: fix pipelines link and remove duplicate text.

### PR DESCRIPTION
**Pull request by miketruty**
docs: fix pipelines link and remove duplicate text.

-pipelines link can point to pipeline anchor.
-style preference: many editors reserve quotation marks for exact spoken/written language.
-remove duplicated line/text
-refine mention of Overview to use Concepts since that's the prominent page title
-add User Guides which is, perhaps, a better reference for walkthroughs

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -22,11 +22,11 @@ Manage your SDLC in Spinnaker using the GUI (graphical user interface), or confi
 - Functions
 
 
-Facilitate GitOps, and grant developers control of provisioning infrastructure for apps. Use Spinnaker’s Clouddriver to deploy to all of the major public cloud providers and Kubernetes. You may even orchestrate configuration and firmware changes as part of Spinnaker delivery https://www.spinnaker.io/concepts/#firewall[pipelines].
+Facilitate GitOps, and grant developers control of provisioning infrastructure for apps. Use Spinnaker’s Clouddriver to deploy to all of the major public cloud providers and Kubernetes. You may even orchestrate configuration and firmware changes as part of Spinnaker delivery https://www.spinnaker.io/concepts/#pipeline[pipelines].
 
 = Why Do I Need Spinnaker?
 
-With Spinnaker, create a “paved road” for application delivery, with guardrails that ensure only valid infrastructure and configuration reach production.
+With Spinnaker, create a *paved road* for application delivery, with guardrails that ensure only valid infrastructure and configuration reach production.
```
Comment (wTylerReid) [inline README.adoc:29]: Nit, but the comma after "application delivery" is unneeded.  I know you didn't change that, but we can polish the grammar while the hood is open :)

File: README.adoc [hunk 2]
```diff
@@ -42,17 +42,17 @@ prebuilt definitions and move changes through environments via a visual interfac
...
 = Using Spinnaker
 
 Spinnaker users should refer to the main
 https://www.spinnaker.io/[Spinnaker site] and https://www.spinnaker.io/setup/[Installation] guide.
 
-For more information on how Spinnaker is designed, see the https://www.spinnaker.io/concepts/[Documentation Overview].
+For more information on how Spinnaker is designed, see the https://www.spinnaker.io/concepts/[Concepts] topic in the documentation Overview.
 
-If you're interested in a detailed walkthrough of the Spinnaker systems, check the https://www.spinnaker.io/reference/[Reference Material].
+If you're interested in a detailed walkthrough of the Spinnaker systems, check the https://spinnaker.io/guides/[User Guides] or the https://www.spinnaker.io/reference/[Reference Material].
```
Comment (wTylerReid) [inline README.adoc:55]: Good call adding the User Guides link.  

In my opinion, we can drop the link to the references page as there isn't much useful info there

**Review by wTylerReid — APPROVED** [review]

File: README.adoc (hunk 1 above)
Comment (miketruty) [reply README.adoc]: Good idea. Done.

File: README.adoc (hunk 2 above)
Comment (miketruty) [reply README.adoc]: Agree. Done.

**Review by miketruty — COMMENTED** [review]
All good comments.

**Review by alivx — APPROVED** [review]

**Review by mattgogerly — APPROVED** [review]

**Pull request merged**